ci-scripts/rhdh-setup/deploy.sh

```diff
@@ -240,6 +246,16 @@ install_rhdh_with_olm() {
     wait_to_start deployment "backstage-developer-hub" 300 300
 }
 
+# shellcheck disable=SC2016,SC1001,SC2086
+psql_debug() {
+    if ${PSQL_LOG}; then
+       oc exec rhdh-postgresql-primary-0 -n "${RHDH_NAMESPACE}" -- sh -c "sed -i "s/^\s*#log_min_duration_statement.*/log_min_duration_statement=${LOG_MIN_DURATION_STATEMENT}/" /var/lib/pgsql/data/userdata/postgresql.conf "
+       oc exec rhdh-postgresql-primary-0 -n "${RHDH_NAMESPACE}" -- sh -c "sed -i "s/^\s*#log_min_duration_sample.*/log_min_duration_sample=${LOG_MIN_DURATION_SAMPLE}/" /var/lib/pgsql/data/userdata/postgresql.conf "
+       oc exec rhdh-postgresql-primary-0 -n "${RHDH_NAMESPACE}" -- sh -c "sed -i "s/^\s*#log_statement_sample_rate.*/log_statement_sample_rate=${LOG_STATEMENT_SAMPLE_RATE}/" /var/lib/pgsql/data/userdata/postgresql.conf "
+    fi
+    oc exec rhdh-postgresql-primary-0 -n "${RHDH_NAMESPACE}" -- sh -c 'pg_ctl -D $PGDATA restart -mf'
```

Comment:
@yogananth-subramanian when installing RHDH with OLM, the name of the DB pod is different - `backstage-psql-developer-hub-0`, please take that into account to support both Helm and OLM